packages/web/src/app/App.tsx

```diff
@@ -48,10 +43,6 @@ export const AppInner = () => {
     <>
       <SomethingWrong />
       <Suspense fallback={null}>
-        <CoinflowPurchaseProtection
```

Comment:
I'm not sure what the implications are of removing this and only rendering it with the modal. I'm also not sure if it's designed to be unmounted and remounted multiple times.

The docs say to add it to "every page on the site": https://docs.coinflow.cash/docs/implement-chargeback-protection

This might significantly change the shape of the data that gets collected and included, so it might throw off the protection algorithm?

Reply:
Yeah I think the idea is to have this always mounted so that the model can collect positive signals about the user through normal app usage. I think moving this to the modal only would reduce the amount of data they get to produce that signal. What's more, the model probably already reflects its current location which means moving it now may mean that future purchases will fail the model's checks and be false positives for chargeback protection?